Assignment-1/code/style.css

```diff
@@ -1 +1,121 @@
+.subdiv
+{
+   width:99vw;
+   height:33vh;
 
+}
+@media screen and (min-width:1025px)
+{
+    .first-div
+    {
+        width:48%;
+        height:98%;
+        background-color:orange;
+        display: inline-block;
+        margin:5px;
+
+
+    }
+    .second-div{
+
+        height:98%;
+        width:98%;
+        background-color: red;
+        margin:5px;
+
+
+    }
+    .third-div
```

Comment:
In This Session(i.e  three divs session)

alignment is not so perfect
the padding was not good in 1024 view

Reply:
you need to concentrate on logic here not on styles you need check based on screen size